src/Grafana/dashboards/aspnetcore.json

```diff
@@ -1270,9 +1281,9 @@
       "type": "table"
     }
   ],
-  "refresh": "10s",
+  "refresh": "1m",
```

Comment:
Changed the default refresh interval to 1m - can change it back. Just had it accidentally left that way when I exported.